index.js

```diff
@@ -1,5 +1,74 @@
 'use strict';
 
```

Comment:
Take a good look, I didn't comment on the `'use strict';`, but on the blank line under that.